.gitignore

```diff
@@ -2,3 +2,4 @@
 .DS_Store
```

Comment:
Please remove this file before we merge this PR

Reply:
Which file?

Reply:
.gitignore